jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/auth/AuthConfig.java

```diff
@@ -60,23 +62,6 @@ public AuthConfig(String username, String password, String email, String auth) {
     this(username, password, email, auth, null);
   }
 
-  public String toHeaderValue(KitLogger logger) {
```

Comment:
This class was completely reformatted and is really hard to track the changes (at least in the GH interface).
Could you please revert the reformatting so that it's clearer for us to see what was changed here?